get.sh

```diff
@@ -248,7 +248,16 @@ getBinaryOpenjdk()
 				echo "curl error code: $download_exit_code"
 				echo "Failed to retrieve $file, exiting. This is what we received of the file and MD5 sum:"
 				ls -ld $file
-				md5sum $file
+				#md5sum $file
```

Comment:
Thanks for your contribution @Niwedita-Rani - were you able to test it locally (perhaps by disconnecting your laptop if you have a Mac during a test run)?

I only have some minor review comments, which is you can remove the 3 commented out lines as they do not add to the understanding of the code

Reply:
Thanks @smlambert for reviewing the changes.I have removed the 3 commented lines.